### PR TITLE
Fix guild not in settings

### PR DESCRIPTION
**Pull request by JoshuaSlui**
This PR ensures that when a server is not present in the settings database, it'll be added and the command will be ran successfully. These types of scenarios tend to happen when the on_guild_join event cannot fire as a result of the bot being offline, or non-responsive.

Additionally requesting @Soapy7261's review on this as well.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -43,6 +43,12 @@ async def on_ready():
 
 @bot.check
 async def block_disabled_commands(ctx):
+    result = (await selector("SELECT config FROM settings WHERE GUILD = %s", [ctx.guild.id]))
+    if result == ():
+        configuration = {'currency': True, 'socials': True}
+        configuration = json.dumps(configuration)
```
Comment (ToothyDev) [inline bot.py:49]: Why not json.dumps that dictionary right away?

**Review by ToothyDev — DISMISSED** [review]

File: bot.py (hunk 1 above)
Comment (JoshuaSlui) [reply bot.py:49]: Because I don't want to do that. This is purely a choice of preference.

**Comment by Soapy7261** [comment]
> Additionally requesting @Soapy7261's review on this as well.

maybe you should add me a reviewer if you want my review

**Comment by JoshuaSlui** [comment]
> > Additionally requesting @Soapy7261's review on this as well.
> 
> maybe you should add me a reviewer if you want my review

accept the fuckin' invite then.

File: bot.py (hunk 1 above)
Comment (Soapy7261) [reply bot.py:49]: I agree with not dumping it right away, as seeing function calls with a long name in a part where inputs should be makes it longer (which is bad) and more ugly

**Comment by Soapy7261** [comment]
Didn't give me one

**Comment by JoshuaSlui** [comment]
There is one for the organization itself?

**Comment by Soapy7261** [comment]
I didn't get an invite 🤷

**Comment by Soapy7261** [comment]
wait im stupid hold on

**Comment by Soapy7261** [comment]
github was being a dumb fuck

**Comment by JoshuaSlui** [comment]
Tell us something new :wheeze:

**Comment by Soapy7261** [comment]
:pensive:

**Review by Soapy7261 — DISMISSED** [review]
this is mainly an sql update so im just guessing this works
if it doesn't im not liable k?

**Comment by JoshuaSlui** [comment]
I mainly wanted your opinion on where i was doing it. This checks it when the command is invoked.

**Comment by Soapy7261** [comment]
im tire

**Comment by Soapy7261** [comment]
also errors should be told to the user instead of just a timeout

**Comment by JoshuaSlui** [comment]
Where are you seeing something about a timeout?

**Comment by Soapy7261** [comment]
timeout

**Comment by Soapy7261** [comment]
dumbass smh

**Comment by JoshuaSlui** [comment]
I have an error handler, 5head. I don't know why that failed.

**Comment by JoshuaSlui** [comment]
I believe because it was a SQL error, that didn't come through the error handler.
Anyway you two should probably redo your review because i just made the `else` clause in the error handler ephemeral and github doesn't care for the necessity of a commit.

**Review by ToothyDev — APPROVED** [review]

**Pull request merged**